src/component/Header/MenuBar/MenuBar.jsx

```diff
@@ -0,0 +1,35 @@
+import { Button, Toolbar } from '@material-ui/core';
+import HomeIcon from '@material-ui/icons/Home';
+import React from 'react';
+import HeaderStyles from '../HeaderStyles';
+import NavItem from './NavItem';
+
+const MenuBar = () => {
+  const classes = HeaderStyles();
+
+  return (
+    <>
+      <Toolbar>
+        <Button color="inherit" className={classes.logo}>
+          <HomeIcon />
+        </Button>
+        <NavItem
+          title="Music"
```

Comment:
don't make it like this, with subtitle, subtitle1, subtitle2... as this is limiting the amount of items we can have. If we decide we need more items later on, then we have to completely change the code. Just pass an array of subitems as the props